app/controllers/api/v1/status_estimations_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Api
+  module V1
+    class StatusEstimationsController < ExternalApiController
+      def create
+        return forbidden unless current_user.admin?
```

Comment:
I think you need to use context[:current_user], otherwise this doesn't work.

Reply:
at least that's how it's working on the ApiController and StatusStoreOwnersController.
if this is using the current_user on the ExternalApiController this might still not work in development, because it fetches the first user, which might not be an admin

Reply:
I can't make it work from Postman.

Reply:
> I think you need to use context[:current_user], otherwise this doesn't work.

That's for jsonapi resources. But I'm going to check it (without the development validations)